gateway/gateway.go

```diff
@@ -1381,23 +1402,23 @@ func (gateway *HandleT) StartWebHandler(ctx context.Context) error {
 	gateway.logger.Infof("Starting in %d", webPort)
 	srvMux := mux.NewRouter()
 	srvMux.Use(headerMiddleware)
```

Comment:
IMO we should have 3 middlewares, with the following order:

1. `statsMiddleware`
3. `activeClientsMiddleware`
2. `headerMiddleware`

This also implies that we shall also clean our handler functions below!

Reply:
made changes with 3 middleware in chain.